addons/website_crm/tests/test_website_crm.py

```diff
@@ -8,30 +8,11 @@
 class TestWebsiteCrm(odoo.tests.HttpCase):
 
     def test_tour(self):
-        self.start_tour("/", 'website_crm_tour')
+        self.start_tour("/", 'website_crm_tour', login='admin')
 
         # check result
         record = self.env['crm.lead'].search([('description', '=', '### TOUR DATA ###')])
         self.assertEqual(len(record), 1)
         self.assertEqual(record.contact_name, 'John Smith')
         self.assertEqual(record.email_from, 'john@smith.com')
         self.assertEqual(record.partner_name, 'Odoo S.A.')
```

Comment:
Why not having done it as we discussed? Filling the form as an admin does not test the same things: you have to modify the form as an admin (first tour), then fill in the form as a public user (second tour).

Reply:
I tried it but it didn't work, after further investigation I figured it was because the change I did during the first tour wasn't applied during the second tour. I now added an extra trigger to ensure the action has changed before we start the second tour and did as we discussed.